mentors/python/g.json

```
@@ -15,4 +15,12 @@
         "avatar_url": null,
         "bio": "I'm enthusiastic learner who is always looking for new projects and challenges and a huge Free Software enthusiast. Reach out if you want to talk about technology, life and sports." 
     }
```

```suggestion
    },
```
A comma needs to be added here.